pkg/pillar/go.mod

```diff
@@ -71,6 +72,7 @@ require (
 	github.com/containerd/stargz-snapshotter/estargz v0.11.4 // indirect
 	github.com/containerd/ttrpc v1.1.0 // indirect
 	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
+	github.com/coreos/yaml v0.0.0-20141224210557-6b16a5714269 // indirect
```

Comment:
I'd much prefer the origin of that fork (https://github.com/go-yaml/yaml) since it has an MIT+Apache license, but I guess this is brought in by github.com/elotl/cloud-init

Reply:
yes, it's a constraint specified [here](https://github.com/elotl/cloud-init/blob/bf3035f6997c8693f050c40366f9eddbfe0b0ceb/Gopkg.toml#L36C1-L38C34). the implementation is however very simple, so we can do our own if needed

Reply:
I think that would be better. Some users are sensitive to additional *GPL licenses (outside of the common ones like the basic Linux kernel, user space, and Xen).

Reply:
done